web_interface/blueprints/api_v3.py

```
@@ -5473,28 +5487,274 @@ def upload_font():
+        # Load font
+        font = None
+        if str(font_path).endswith('.bdf'):
+            # For BDF fonts, try using freetype via PIL BDF support or fallback
+            try:
+                import freetype
+                face = freetype.Face(str(font_path))
+                face.set_pixel_sizes(0, size)
+                # For BDF, we'll render character by character
+                # This is a simplified approach - full BDF rendering is complex
+                font = ImageFont.load_default()
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)